doc/tutorial/fcm_fundamentals

```diff
@@ -0,0 +1,859 @@
+            <br/>
+            <div class="panel-group" id="accordion">
+                <div class="panel panel-default">
+                    <div class="panel-heading">
+                        <h4 class="panel-title">
+                            <a data-toggle="collapse" data-parent="#accordion" href="#collapseOne">
+                            <span class="glyphicon"></span>
+                            What other pros and cons can you think of for each of the ways of working?
+                            </a>
+                        </h4>
+                    </div>
+                    <div id="collapseOne" class="panel-collapse collapse">
+                        <div class="panel-body">
+                            Nicholas and Danny pros
+                            <br/>
+                            There will be no chance of conflicts.
+                            <br/>
+                            Nicholas and Danny cons
+                            <br/>
+                            Someone may make a change that the first person does not agree with.
+                            <br/>
+                            The code will take longer to develop.
+                            <br/>
+                            Changes cannot be tracked as easy.
+                            <br/>
+                            Dorris pros
+                            <br/>
+                            Simultaneous code development, therefore speeding up the project.
+                            <br/>
+                            Full version controlled, with who did what when.
+                            <br/>
+                            Fully traceable.
+                            <br/>
+                            Dorris cons
+                            <br/>They will have do manually resolve any conflicts on merging, however this is the same as Nicholas and Danny's suggestion except here they will be able to see how their code changes interact with each other.
+                            This is unlike Nicholas and Danny's suggestion where they will make changes pass the code on and then will probably have to pass around a few times to agree on the code changes.
+                            <br/>
+                            Did you get all the reasons above and think of others?
+                        </div>
+                    </div>
+                </div>
+                <br/>
+                Scenario 2
+                <br/>Daisy has been developing some code, she now wants help from Mike to develop the code further using Mikes expertise.
+                <br/>
+                Mike has taken a copy of the code and Daisy is still developing it.
+                <br/>
+                Mike tries to merge his code in to the trunk version that Daisy has committed but the code has been updated and there is a conflict.
+                <br/>
+                Using Fcm they can see the conflict and resolve it using a built in tool.
```

Comment:
Needs highlighting that FCM helps them, perhaps Benefit of FCM: using FCM they can see...